core/controllers/reader.py

```diff
@@ -1016,3 +1013,25 @@ def post(self, exploration_id):
             self.payload.get('report_text'),
             self.user_id)
         self.render_json(self.values)
+
+
+class ExplorationFeaturesHandler(base.BaseHandler):
```

Comment:
A doubt here... IS_IMPROVEMENTS_TAB_ENABLED would be used in the editor view only right? and is_playthrough_recording_enabled would be used in the player view only right? So I am not sure if logically this code should reside in the editor.py or the reader.py file. 

Maybe we should keep it as 2 handlers and place them in their respective files? 

If I got the logic wrong, could you please clarify? Thanks!

EDIT: After reading the complete PR, I am kind of confused about the placement of this handler (I don't think it was right before either). This handler seems to cater to both views and is not specific to either of them. Just like how the frontend service is sitting in the common folder. Can we do something similar here? What's your opinion on this?

Reply:
Done, I've introduced the new module "features"  +tests.